src/plugins/intel_gna/src/common/gna_target.cpp

```diff
@@ -44,8 +44,8 @@ static const std::unordered_map<Gna2DeviceVersion, DeviceVersion> GnaDeviceMap{
     {Gna2DeviceVersionEmbedded3_1, DeviceVersion::GNA3_1},
     {Gna2DeviceVersion3_5, DeviceVersion::GNA3_5},
     {Gna2DeviceVersionEmbedded3_5, DeviceVersion::GNAEmbedded3_5},
-    {Gna2DeviceVersionEmbedded3_5, DeviceVersion::GNA3_6},
-    {Gna2DeviceVersionEmbedded3_5, DeviceVersion::GNA4_0},
+    {Gna2DeviceVersionEmbedded3_6, DeviceVersion::GNA3_6},
+    {Gna2DeviceVersionEmbedded3_6, DeviceVersion::GNA4_0},
```

Comment:
please decide on either

Reply:
Could you elaborate? Decide on either what?

Reply:
due to the same key, the second entry is dead right?

Reply:
It's not dead, it means mapping 4.0 <-> 3.6. Please see how this map is used.

Reply:
when you iterate over GnaDeviceMap do you see Gna2DeviceVersionEmbedded3_6 -> DeviceVersion::GNA4_0 mapping present?
additionally, does the following code throw or not?:

          DeviceToGna(DeviceVersion::GNA4_0);

Reply:
do you want the second dead entry for documentation purposes? IF yes, then for not confuse the readers, please say it out explicitly - prepend the line with following comment as minimum:

             // for documentation purposes:

other, more meaningful comments would be welcome of course